src/components/layout.js

```diff
@@ -40,13 +40,42 @@ class Layout extends React.Component {
       defaultClientAuth.apiKeyPrefix = this.props.user.auth.scheme;
       defaultClientAuth.apiKey = this.props.user.auth.token;
 
+      // This is the first component that loads,
+      // and refreshUserInfo and the subsequent organisationsLoad() are the
+      // first calls happa makes to the API.
       this.props.actions
         .refreshUserInfo()
         .then(() => {
           this.props.dispatch(organizationsLoad());
           return null;
         })
         .catch(error => {
+          if (error.status === 401) {
+            this.props.flashActions.flashAdd({
+              message:
+                'Please log in again, your previously saved credentials appear to be invalid.',
+              class: 'warning',
+            });
+
+            this.props.dispatch(push('/login'));
+          } else {
+            this.props.flashActions.flashAdd({
+              message: (
+                <div>
+                  <strong>
+                    Something went wrong while trying to load user and
+                    organization information.
+                  </strong>
+                  <br />
+                  {error.body
+                    ? error.body.status_text
+                    : 'Perhaps our servers are down, please try again later or contact support: support@giantswarm.io'}
```

Comment:
I would like to say something else. That message here implies the worst unproven assumption possible. Some more positive wording would be good. 
```suggestion
                    : 'In case the connection issue persists, please try again later or contact support: support@giantswarm.io'}
```

Reply:
ah indeed... 

How about we go for just "Please try again later or contact support: support@giantswarm.io"